src/librustc_trans/mir/block.rs

```diff
@@ -116,6 +116,9 @@ impl<'bcx, 'tcx> MirContext<'bcx, 'tcx> {
                 if let Some(cleanup_pad) = cleanup_pad {
                     bcx.cleanup_ret(cleanup_pad, None);
                 } else {
+                    let llpersonality = bcx.fcx().eh_personality();
+                    bcx.set_personality_fn(llpersonality);
```

Comment:
> I said already in #36895, 

Apologies; I wasn't CCed on that PR, so didn't know there were already reviews :-)